packages/webpack-exp-plugin/index.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const { NormalModuleReplacementPlugin } = require('webpack')
+const replacerToExp = require('./replacer')
+
+class LegoExpPlugin extends NormalModuleReplacementPlugin {
+  constructor(experiments = {}, packageName = '@yandex-lego/components') {
```

Comment:
Давай уберем этот фолбек, чтобы сделать абстрактно (возможно стоит поменять параметры местами):

```suggestion
  constructor(experiments = {}, packageName) {
```

```ts
new WebpackExpPlugin('@yandex/ui', {
  Button: 'scoped-css-exp',
})
```

Reply:
Ну и можно добавить сразу валидацию + юнит:

```js
if (packageName === undefined) {
  throw new Error('...')
}
```